src/router.js

```diff
@@ -296,6 +296,12 @@ const routes = (
                 component={LoadableComponent({ loader: () => import('./OrganisationUnitHierarchy') })}
                 onEnter={initStateOuHierarchy}
             />
+            <Route
+                path="sqlViews/:modelId"
+                component={LoadableComponent({ loader: () => import('./SqlView') })}
```

Comment:
Can also use the LoadableWithLoaders, if you want to load something before the component is mounted.